SignalServiceKit/src/Contacts/SignalAccount.m

```diff
@@ -50,16 +50,9 @@ - (nullable NSString *)uniqueId
     return _recipientId;
 }
 
-- (NSString *)displayName
```

Comment:
The only place we used this was when determining whether to show the profile name, so we should return nil if there is no local contact name.